README.md

```diff
@@ -14,7 +14,7 @@
 RedisBloom adds a set of probabilistic data structures to Redis, including Bloom filter, Cuckoo filter, Count-min sketch, Top-K, and t-digest. Using this capability, you can query streaming data without needing to store all the elements of the stream. Probabilistic data structures each answer the following questions:
 
 - Bloom filter and Cuckoo filter:
-  -  Did value _v_ already appear in the data stream?
+  -  Did value _v_ already appear in the data stream
```

Comment:
Why did you remove the question mark?

Reply:
This is just a dummy pr to check if CI works fine.